PrioritizationModel/AssetPrioritizationModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using PrioritizationService.DTOs;
+
+namespace PrioritizationModel;
+
+internal class AssetPrioritizationModel : IPrioritizationModel
```

Comment:
Change the name of the class (probably in visual studio so it doesnt get messed up) - probably to just "PriorizitationModel" or something else that shows that it is an intial version of it

Reply:
Approved with: Commit [f1d3b7c](https://github.com/diversiwaver/aFRR-Prioritization-Service/pull/9/commits/f1d3b7c7585fe1528482d72144d5651a4d2490ef)